source/UF/Pushouts.lagda

```diff
@@ -0,0 +1,418 @@
+Ian Ray, 15th January 2025
+
+Edited by Ian Ray on 16th March 2025.
+
+Pushouts are defined as a higher inductive type (in the form of a record type). We
+postulate point and path constructors and the (dependent) universal property.
+We will derive other important results like induction and recursion principles along
+and the corresponding propositional computation rules. Of course, due to Kristina
+Sojakova's dissertation (and the following paper on the same topic doi:
```

Comment:
There is a missing verb in the sentence starting "Of course, due to".

Reply:
Not sure I follow. It may sounds a bit clunky with the mid sentence parthetical and reference link, but the sentence is simply: Of course, due to Sojakova's dissertation, it is well known that for higher inductive
types with propositional computation rules the following are equivalent:
*list follows*.